src/utils/auth.js

```diff
@@ -1,8 +1,18 @@
+// Helper function to get a specific cookie value by name
+export const getCookieValue = (name) => {
+  if (typeof document !== "undefined") {
+    const value = document.cookie
+      .split("; ")
+      .find((row) => row.startsWith(`${name}=`))
+      ?.split("=")[1];
+    return value ? decodeURIComponent(value) : null;
+  }
+  return null;
+};
+
+// Updated isAuthenticated function using the helper
 export const isAuthenticated = () => {
-    if (typeof window !== "undefined") {
```

Comment:
@Shryansh107 let's not change the login for verifying if user is authenticated for now, it can introduce bugs in system.